app/views/layouts/_header.html.erb

```diff
@@ -181,7 +181,7 @@
     <li><a href="/people">Contributors</a></li>
 	           <li class="divider"></li>
 	       <% end %>
-              <li class="dropdown">
+              <ul class="dropdown">
```

Comment:
Hi, I think perhaps it has to be a `<ul>` inside of a `<li>` -- can you try that and also upload a screenshot of how it looks? You may want to test it out on a narrow window (to see how it'll look on a smartphone, too) as well.

Thank you very much!